template/Host.cs.ejs

```diff
@@ -68,30 +68,30 @@ public class Host
     <% }) %>
 
     <% schema.imports.forEach(imp => { %>
-    [DllImport("extism", EntryPoint = "<%= imp.name %>")]
-    private static extern <%= imp.output ? "ulong" : "void" %> <%= imp.name %>Impl(<%= imp.input ? "ulong input" : "" %>);
+    [DllImport("extism", EntryPoint = "<%- imp.name %>")]
+    private static extern <%- imp.output ? "ulong" : "void" %> <%- imp.name %>Impl(<%- imp.input ? "ulong input" : "" %>);
 
     <% if (needsDocumentation(imp)) { %>
         /// <summary>  <% -%>
         <% imp.description.split('\n').filter(line => line.trim() !== "").forEach(line => { %>
-            /// <%= line.trim() -%>
+            /// <%- line.trim() -%>
         <% }) %>
         /// </summary>
         <% if (imp.input && imp.input.description) { -%>
-        /// <param name="input"><%= imp.input.description %></param>
+        /// <param name="input"><%- imp.input.description %></param>
         <% } -%>
         <% if (imp.output && imp.output.description) { -%>
-        /// <returns><%= imp.output.description %></returns>
+        /// <returns><%- imp.output.description %></returns>
```

Comment:
@mhmd-azeez i'd leave it to you to say if we should actually leave `<%=` here since this is XML. I'm not sure what it expects.

Reply:
The characters are expected to be escaped, otherwise they won't get rendered properly